opentelemetry_collector/receiver/dockerstats/scraper.go

```diff
@@ -41,6 +41,20 @@ var (
 		Type:        mpb.MetricDescriptor_GAUGE_INT64,
 		LabelKeys:   []*mpb.LabelKey{containerNameLabel},
 	}
+	nwRecvBytesDesc = &mpb.MetricDescriptor{
+		Name:        "container/network/received_bytes",
+		Description: "Bytes received by container over all network interfaces",
+		Unit:        "byte",
```

Comment:
This should be "bytes" not "byte" to match the list of well-known streamz units.

Reply:
It's byte.

See https://pkg.go.dev/github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1?tab=doc#MetricDescriptor and http://unitsofmeasure.org/ucum.html#section-Prefixes-and-Units-Used-in-Information-Technology.